feat(changelog): elastic-metal-deprecated-legacy-oses-depreciation 2026-07-29 (#6739)
scaleway/docs-content: changelog/july2026/2026-07-29-elastic-metal-deprecated-legacy-oses-depreciation.mdx

---

> title: Legacy OSes depreciation

Reviewer: ```suggestion
title: Deprecation of legacy operating systems
```

status: deprecated
date: 2026-07-29
category: bare-metal
product: elastic-metal
---

> Effective October 1st, support for several operating systems will be officially deprecated due to reaching their End of Life (EOL). 

Reviewer: ```suggestion
Effective October 1st, 2026, support for several operating systems will be officially deprecated because they have reached their End of Life (EOL). 
```

> Below is the list of affected OS versions along with their respective EOL dates:

Reviewer: ```suggestion
The following are the affected OS versions and their EOL dates:
```

- Ubuntu 20.04: EOL May 2025
- Debian 10: EOL June 2024
- Debian 11: EOL August 2026
- Proxmox 7: EOL July 2024
- Proxmox 8: EOL August 2026
- CentOS 7: EOL June 2024
- Windows Server 2019: EOL September 2024

> We recommend upgrading your infrastructure to supported OS versions ahead of the deadline to ensure continued security and system stability.

Reviewer: ```suggestion
Scaleway recommends upgrading your infrastructure to supported OS versions ahead of the deadline to ensure continued security and system stability.
```
